package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "productcat-widgets-javascript",
+  "version": "1.0.0",
+  "description": "A collection of javascript widgets to instantly add productcat search functionality into websites",
+  "main": "index.js",
+  "directories": {
+    "lib": "common"
+  },
+  "scripts": {
+    "start": "webpack-dev-server --env tracking --config webpack.dev.js",
+    "build-prod": "webpack --env tracking --config webpack.prod.js",
+    "test": "mocha",
+    "get-version": "echo $npm_package_version"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/visenze/productcat-widget-javascript"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@analytics/google-analytics": "^0.5.0",
```

Comment:
why do we need this? pls remove all dependencies that we do not use.

Reply:
removed